nextcord/application_command.py

```diff
@@ -1812,6 +1813,11 @@ def __init__(
             Permission(s) required to use the command. Inputting ``8`` or ``Permissions(administrator=True)`` for
             example will only allow Administrators to use the command. If set to 0, nobody will be able to use it by
             default. Server owners CAN override the permission requirements.
+        nsfw: :class:`bool`
+            If the command should be usable in NSFW channels or not. Setting to ``True`` will disable the command from
+            being usable in Non NSFW channels.
+
+            .. versionadded:: 2.1
```

Comment:
```suggestion
            .. versionadded:: 2.4
```
I don't think this PR will be able to make it in 2.3

Reply:
It isn't even guaranteed discord will release this before 2.4.